panacea/rpc_client.go

```diff
@@ -0,0 +1,113 @@
+package panacea
+
+import (
+	"context"
+	"fmt"
+	ics23 "github.com/confio/ics23/go"
+	"github.com/cosmos/ibc-go/v2/modules/core/23-commitment/types"
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/light"
+	"github.com/tendermint/tendermint/light/provider"
+	"github.com/tendermint/tendermint/light/provider/http"
+	dbs "github.com/tendermint/tendermint/light/store/db"
+	"github.com/tendermint/tendermint/rpc/client"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	dbm "github.com/tendermint/tm-db"
+	"time"
+)
+
+const blockPeriod = 6 * time.Second
+
+type RpcClient struct {
+	RpcClient   *rpchttp.HTTP
+	LightClient *light.Client
+}
+
+// NewRpcClient set RpcClient with trustedBlock Info and returns, if successful,
+// a RpcClient that can be used to query data with light client verification & merkle proof.
+// Period of trustOptions can be changed according to slashing period.
+func NewRpcClient(ctx context.Context, chainID, rpcAddr string, trustedHeight int, trustedBlockHash []byte) (*RpcClient, error) {
+	rpcClient, err := rpchttp.New(rpcAddr, "/websocket")
+	if err != nil {
+		return nil, err
+	}
+
+	trustOptions := light.TrustOptions{
+		Period: 365 * 24 * time.Hour,
```

Comment:
I think we should change this value to 2 weeks.
But first let's leave it like this

Reply:
I agree. After add some logics (get initial trusted block info from user input, automatic update trusted block info from panacea), we can change this value to 2 weeks. 
For now, I set it to 2 years for testing.